backend/lib/components/Geo/gmaps/getDocFacilities.js

```diff
@@ -0,0 +1,20 @@
+const {Client} = require("@googlemaps/google-maps-services-js");
+const {
+    config: { geo },
+  } = require("../../../../config");
+  
+var places = [];
+
+const getDoctorFacilities = async (location, maxFacilities) => { 
+    const client = new Client({});
+    await client.placesNearby({ params: { location: location, type: 'doctor', rankby: 'distance', key: geo.googleMapsApiKey, }
+    })
```

Comment:
```suggestion
const getDoctorFacilities = async (location, maxFacilities) => { 
    let places = [];
    const client = new Client({});
    await client.placesNearby({ params: { location: location, type: 'doctor', rankby: 'distance', key: geo.googleMapsApiKey, }
    })
```

Reply:
Would rather not use let since the block script could be too strict in this case.   Would like to leave this as var. Here's a good discussion.   https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var
Look at "Scoping Rules" down the page that got 6307 upvotes.  
Probably in this case, let would be too strict for scoping.

Reply:
Ohh it should be fine, the variable places is not being used anywhere else so this suggestion should work fine. And we need to avoid as much as possible using `var`...

Reply:
Ok - I see what you mean - I was afraid of underscoping, and you would rather not overscope...So I'm changing all the vars to lets.

Reply:
@maskowe I wouldn't mix the use of await & then/catch here. I'd try to be consistent with async/await like the rest of the app - for consistency at least but it's also more readable. This could be re-written (for example):

```javascript
const { data }  = await client.placesNearby({ params: { location, type: 'doctor', rankby: 'distance', key: geo.googleMapsApiKey });
return data.results.length > 0 ? data.results.slice(0, maxFacilities) data.results;
}
```

Reply:
But let's wait til we resolve the question about which type to use. I don't think we'll need to this file in any case. 

I do think there will be a few other eslint issues here. Unfortunately it's not working with docker right now (open PR I need to review #1600).